src/leases/components/LeasePage.js

```diff
@@ -848,10 +850,18 @@ class LeasePage extends Component<Props, State> {
       leaseAttributes,
       leaseMethods,
       match: {params: {leaseId}},
+      userActiveServiceUnit,
       usersPermissions,
     } = this.props;
     const areFormsValid = this.validateForms();
 
+    const isServiceUnitSameAsActiveServiceUnit = () => {
+      if (userActiveServiceUnit && currentLease.service_unit) {
```

Comment:
Same comment as for the other file. If userActiveServiceUnit cannot be undefined, simplify this check with optional chaining.